backend/models/generic.go

```diff
@@ -6,7 +6,7 @@ import (
 )
 
 type Common struct {
-	ID        uint           `gorm:"primaryKey" json:"id"`
+	Id        uint           `gorm:"primaryKey" json:"id"`
```

Comment:
After reading https://gorm.io/docs/conventions.html, I understand why you left it as ID. I'm ok using `ID` instead of `Id` if gorm only recognizes `ID` as the default primary key. I believe this will make implementation of foreign keys easier.

Reply:
In which case we could delete `gorm:"primaryKey"`

Reply:
Nevermind, it doesn't care about cases, i made the other change in https://github.com/lynx-locks/lynx-web/pull/19